ray-operator/controllers/ray/rayservice_controller.go

```diff
@@ -243,6 +243,7 @@ func (r *RayServiceReconciler) calculateStatus(ctx context.Context, rayServiceIn
 	rayServiceInstance.Status.ActiveServiceStatus.Applications = activeClusterServeApplications
 	rayServiceInstance.Status.PendingServiceStatus.Applications = pendingClusterServeApplications
 
+	var servingClusterName string
```

Comment:
Maybe use a boolean to indicate whether the services point to the pending cluster instead?

Reply:
Done, but I think using a Boolean feels like leaving the actual check outside of the shouldXXX function.